src/debugAdapter/types/@truffle/environment.d.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Consensys Software Inc. All rights reserved.
+// Licensed under the MIT license.
+
+declare module '@truffle/environment' {
```

Comment:
are there no types on the environment package?

Reply:
Not yet, [`@truffle/environment`](https://github.com/trufflesuite/truffle/tree/develop/packages/environment) uses pure JS. Not sure if in the future they have plans to migrate or at least include type definitions.